modules/virt-downloading-virtctl-client.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// virt/install/virt-enabling-virtctl.adoc
+
+[id="virt-downloading-virtctl-client_{context}"]
+= Downloading the virtctl client
+
+Download the `virtctl` client with or without an internet connection using the client provided in a pod in the cluster.
+
+.Prerequisites
+
+* You must have an activated {product-title} subscription to access the download page on the Customer Portal.
```

Comment:
This is not necessary anymore. The product must be installed to be able to download the client.